src/main/java/uk/co/onsdigital/job/model/CreateJobRequest.java

```diff
@@ -35,8 +35,8 @@
      *
      * @return the sorted dimensions from the request.
      */
-    public SortedMap<String, SortedSet<String>> getSortedDimensionFilters() {
-        final SortedMap<String, SortedSet<String>> result = new TreeMap<>();
+    public Map<String, Set<String>> getSortedDimensionFilters() {
```

Comment:
Any reason for this change? The filename generation relies on the dimensions and options being sorted, so the type was enforcing that. I'd rather we kept that type if we can. Otherwise, please add a unit test to ensure the result is actually sorted.